src/client/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -67,8 +67,9 @@ public interface IAppConfig
         IDictionary<string, string> ExtraQueryParameters { get; }
 
         /// <summary>
+        /// Indicates whether or not the current application object is configured to use brokered authentication.
         /// </summary>
-        bool IsBrokerEnabled { get; }
+        bool IsBrokerConfiguredByUser { get; }
```

Comment:
>IsBrokerConfiguredByUser [](start = 13, length = 24)

nit: idk about this renaming. mainly  because the "user" does not configure the broker, the app developer does. So it's slightly misleading. @bogavril@microsoft.com do you have any ideas?

Reply:
`IsBrokerConfigured` is probably enough ...